sound/soc/sof/intel/hda.h

```diff
@@ -17,9 +17,13 @@
 #define PCI_TCSEL			0x44
 #define PCI_CGCTL			0x48
 
+/* PCI_TVSEL bits */
+#define PCI_TCSEL_ADSPPGD               BIT(2)
+
 /* PCI_CGCTL bits */
 #define PCI_CGCTL_MISCBDCGE_MASK	BIT(6)
 #define PCI_CGCTL_LSRMD_MASK		BIT(4)
```

Comment:
this bit should be PGCTL one, rename and move it to group /* PCI_PGCTL bits */

Reply:
sure, that will avoid confusion!